README.md

```diff
@@ -177,37 +180,36 @@ Steppers can exert damaging forces, even if they are moving slow. If in doubt, s
+Read and control the digital, and analog, input/output pins of the target device via I2C. Can replace a dedicated digital or analog port expander like MCP23017, PCF8574, PCF8591, or ADS1115. Can be used like  the plain Arduino `digitalRead()` etc. commands. See [`Pin_control.ino`](examples/Pin_control/Pin_control.ino) example.
 
 ## ESP32sensorsI2C
 
-Read an ESP32's touch sensors, hall sensor, and (if it works) temperature sensor via I2C. Can use the optional I2Cwrapper interrupt mechanism to inform the controller about a touch button press. See `ESP32sensors.ino` example.
+Read an ESP32's touch sensors, hall sensor, and (if it works) temperature sensor via I2C. Can use the optional I2Cwrapper interrupt mechanism to inform the controller about a touch button press. See [`ESP32sensors.ino`](examples/ESP32sensors/ESP32sensors.ino) example.
 
-# How to add new modules
+# How to Add New Modules
 
-You can use the templates provided in the templates subfolder if you want to add your own modules and **implement your own I2C target device**.
+You can use the templates provided in the [`templates` subfolder](templates) if you want to add your own modules and **implement your own I2C target device**.
 
 - `template_I2C.h`  and `template_I2C.cpp` - controller library templates. Their main function is to define an interface for the target's functionality and the related command codes (see [limitations](#Limitations for module authors)). Each function is implemented so that the function's command code and parameters are transmitted to the target with the help of the I2Cwrapper library.
   - Often, the header file `template_I2C.h`  will very closely resemble the header file of the library that you are addressing on the target device's side.
   - The implementation  `template_I2C.cpp`, however, looks quite different: It will do nothing else but "wrap" each function's arguments into a command, transmit it to the target, and, optionally, receive the target device's reply.
-- `template_I2C.h`  and `template_I2C.cpp` - controller library templates. Its main function is to define an interface for the target's functionality. Each function is implemented so that the function's command code and parameters are transmitted to the target with the help of the I2Cwrapper library.
 - `template_I2C_firmware.h` - Target firmware templates. Here, the most important part is injecting code into the command interpreter that will "unwrap" the controller function's command codes, react adequately, and, optionally, prepare a reply.
 
-Refer to the documentation within the templates' source code and to the existing modules for more details and illustration.
+Refer to the documentation within the templates' source code and to the [existing modules](src) for more details and illustration.
 
-### A note on messages and "(sub)units"
+### A Note on Messages and "(Sub)Units"
```

Comment:
What would APA say about "(X)Y" capitalization?

Reply:
Not sure, I think it's safe to pick a variant that looks consistent with the other headings.